e2e_tests/tests/task/test_generic_tasks.py

```diff
@@ -0,0 +1,190 @@
+                forkedFrom=task_resp.taskId,
+                parentId=None,
+                inheritContext=False,
+                noPause=False,
+            )
+            fork_task_resp = bindings.post_CreateGenericTask(test_session, body=req)
+
+            # Get fork task Config
+            command = ["det", "task", "config", fork_task_resp.taskId]
+
+            res = subprocess.run(
+                command, universal_newlines=True, stdout=subprocess.PIPE, check=True
+            )
+            result_config = util.yaml_safe_load(res.stdout)
+            expected_config = {"entrypoint": ["python3", "run_fork.py"]}
+            assert expected_config == result_config
+
+
+@pytest.mark.e2e_cpu
+def test_kill_generic_task() -> None:
+    """
+    Start a simple task without a context directory and grab its config
+    """
+    test_session = api_utils.determined_test_session()
+
+    with open(conf.fixtures_path("generic_task/test_config.yaml"), "r") as config_file:
+        # Create task
+        config = ntsc.parse_config(config_file, None, [], [])
+        config_text = util.yaml_safe_dump(config)
+        context_directory = context.read_v1_context(
+            pathlib.Path(conf.fixtures_path("generic_task")), []
+        )
+
+        req = bindings.v1CreateGenericTaskRequest(
+            config=config_text,
+            contextDirectory=context_directory,
+            projectId=None,
+            forkedFrom=None,
+            parentId=None,
+            inheritContext=False,
+            noPause=False,
+        )
+        task_resp = bindings.post_CreateGenericTask(test_session, body=req)
+
+        # Kill task
+        command = ["det", "task", "kill", task_resp.taskId]
+
+        subprocess.run(command, universal_newlines=True, stdout=subprocess.PIPE, check=True)
+
+
+@pytest.mark.e2e_cpu
+def test_pause_and_unpause_generic_task() -> None:
+    """
+    Start a simple task without a context directory and grab its config
+    """
+    test_session = api_utils.determined_test_session()
+
+    with open(conf.fixtures_path("generic_task/test_config.yaml"), "r") as config_file:
+        # Create task
+        config = ntsc.parse_config(config_file, None, [], [])
+        config_text = util.yaml_safe_dump(config)
+        context_directory = context.read_v1_context(
+            pathlib.Path(conf.fixtures_path("generic_task")), []
+        )
+
+        req = bindings.v1CreateGenericTaskRequest(
+            config=config_text,
+            contextDirectory=context_directory,
+            projectId=None,
+            forkedFrom=None,
+            parentId=None,
+            inheritContext=False,
+            noPause=False,
+        )
+        task_resp = bindings.post_CreateGenericTask(test_session, body=req)
+
+        # Pause task
+        command = ["det", "task", "pause", task_resp.taskId]
+
+        subprocess.run(command, universal_newlines=True, stdout=subprocess.PIPE, check=True)
+
+        # Pause task
```

Comment:
similar to the `kill` test above, I think it'd be useful to check if the task has been paused and unpaused.